lib/devise/controllers/helpers.rb

```diff
@@ -36,11 +36,11 @@ module ClassMethods
         #     before_action ->{ authenticate_blogger! :admin }  # Redirects to the admin login page
         #     current_blogger :user                             # Preferably returns a User if one is signed in
         #
-        def devise_group(group_name, opts={})
+        def devise_group(group_name, opts = {})
           mappings = "[#{ opts[:contains].map { |m| ":#{m}" }.join(',') }]"
 
           class_eval <<-METHODS, __FILE__, __LINE__ + 1
-            def authenticate_#{group_name}!(favorite=nil, opts={})
+            def authenticate_#{group_name}!(favorite=nil, opts = {})
```

Comment:
Another arg here that could use some additional spacing around `=` :)

Reply:
Have done all revision!
Could you confirm the commit?

Reply:
:+1: Merged, thanks!